pkg/controller/core/clusterqueue_controller.go

```diff
@@ -160,6 +160,8 @@ func (r *ClusterQueueReconciler) Create(e event.CreateEvent) bool {
 		// No need to interact with the cache for other objects.
 		return true
 	}
+	defer r.notifyWatchers(nil, cq)
```

Comment:
make sure existing watchers ignore create events if they didn't need them before

Reply:
I confirmed that the existing watcher is only for resourceFlavors, and it skips to send a message when it receives create events.

https://github.com/kubernetes-sigs/kueue/blob/f3267deaa2ae65f37675878dfeb679a97cb47d80/pkg/controller/core/resourceflavor_controller.go#L182-L203